cypress/e2e/createProject.cy.ts

```diff
@@ -8,6 +8,7 @@ const createProjectModalSelector = '[data-test="create-project-modal"]';
 
 describe('Create Project', () => {
   beforeEach(() => {
+    cy.login();
```

Comment:
Should we repeat this line everywhere? I suggest to have a auth provider and use it in mount custom command.

Reply:
we might not need to do login in some tests
so having it globally enabled for every test won't work
or we can like enable it somehow for all test and do a logout specifically for tests that do not require it
but that's not also a good idea IMO

Reply:
Our app mostly provides us protected routes.
in many cases we need `cy.login()` . so what's ur idea?

Reply:
no idea really 
it's a thing that we should do, w should always login to do something 
in any other case it would mean we would have to have a dependency to prior test which is a bad practice

Reply:
Let's discuss this in the other thread.